segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/WriteOperation.java

```diff
@@ -157,7 +154,7 @@ private Object handleException(Throwable e) {
         throw new CompletionException(ex);
     }
 
-    private Object postCommit() {
+    private Void postCommit() {
```

Comment:
These changes seem to be unrelated with delayed future cancelation..

Reply:
Reverted